src/components/SkillRatingCard/SkillRatingCard.js

```diff
@@ -39,7 +54,7 @@ class SkillRatingCard extends Component {
 
   updateWindowDimensions = () => {
     this.setState({
-      width: window.innerWidth * 0.8 > 500 ? 400 : window.innerWidth * 0.8,
+      chartWidth: window.innerWidth * 0.8 > 500 ? 400 : window.innerWidth * 0.8,
```

Comment:
Store `window.innerWidth` in a const variable and then check. We don't want to write same varibale name again and again.